### PR TITLE
Cgo#11

### PR DESCRIPTION
**Pull request by MoreWaqar**
Completed Issue #11 
- Highlighted all SGW buildings per the Issue requirement
- Created a placeholder function to aid further development. Function provides building code of a tapped highlighted polygon. Currently printed to console.
- Added a boolean value in the GoogleMaps widget that can flip the polygon visibility off if need be.

### CODE REVIEW

File: lib/utilities/ConcordiaConstants.dart [hunk 1]
```diff
@@ -34,6 +34,241 @@ const String FG_BUILDING_INITIAL = "FG";
...
+    'xcoords': [45.496533, 45.496541, 45.496621, 45.496672],
+    'ycoords': [-73.586209, -73.586041, -73.585972, -73.586086],
+  },
+  {
+    'Building': 'T Annex',
+    'code': 'T',
+    'xcoords': [45.496665, 45.496742, 45.496711, 45.496631],
+    'ycoords': [-73.579336, -73.579256, -73.579189, -73.579269],
+  },
+  {
```
Comment (DBatistaFerreira) [inline lib/utilities/ConcordiaConstants.dart:246]: Duplicates for TD?

File: lib/widgets/component/GoogleMapsComponent.dart [hunk 2]
```diff
@@ -16,24 +15,50 @@ class GoogleMapsComponent extends StatefulWidget {
 class GoogleMapsComponentState extends State<GoogleMapsComponent> {
   Completer<GoogleMapController> _controller = Completer();
   String currentCampus = 'SGW';
+  bool polygonVisibility = true;
+
+  void _infoPanel(String buildingCode) {
+    print(buildingCode);
+    // This method is triggered when a polygon is clicked. Currently it only prints the building code for the building you tap
+  }
+
```
Comment (DBatistaFerreira) [inline lib/widgets/component/GoogleMapsComponent.dart:24]: Awesome way of doing this :)

**Review by DBatistaFerreira — COMMENTED** [review]
Any way we can implement unit tests for this feature? @MoreWaqar

File: lib/utilities/ConcordiaConstants.dart (hunk 1 above)
Comment (MoreWaqar) [reply lib/utilities/ConcordiaConstants.dart:246]: Updated the file to remove that duplicate.

**Comment by MoreWaqar** [comment]
@Pwnagebird Not entirely sure how a test for the polygon functionality could be instituted. However, the method that'll be triggered should be the target of testing once a pop up page is built.

**Comment by neliop** [comment]
We can have a unit test to count the number of polygons on the map to verify they are there. And then of course test the on tap functionality like waqar said.

**Review by DBatistaFerreira — APPROVED** [review]
Absolutely amazing work Waqar!

**Review by neliop — APPROVED** [review]
Approved.

**Comment by tsantalis** [comment]
I am impressed! Codecov, Travis CI, code reviews

**Pull request merged**